content/en/docs/tools-libraries/libraries.md

```diff
@@ -22,6 +22,11 @@ toc: true
 
 - [SimpleWebAuthn](https://simplewebauthn.dev/) (Matthew Miller)
 
+### Java
```

Comment:
Let me know if you want the headings in any particular order, I just appended from the bottom.